openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -166,7 +166,7 @@ def openc3_script_sleep(sleep_time = nil)
         return false
       end
 
-      def display_screen(target_name, screen_name, x = nil, y = nil, scope: $openc3_scope)
+      def display_screen(target_name, screen_name, x = nil, y = nil, scope: RunningScript.instance.scope)
```

Comment:
why scope change?

Reply:
I don't think it was correct. If you look at the rest of the file the scope should be coming from the `RunningScript.instance.scope`. It can still be manually overridden but I think that's the correct default.